tests/phpunit/tests/admin/includesMedia.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/**
+ * @group media
+ * @group admin
+ */
+class Tests_Admin_includesMedia extends WP_UnitTestCase {
+	/** @var \WP_Post */
+	protected static $post;
+	/** @var \WP_Post */
+	protected static $different_post;
+
+	public static function wpSetUpBeforeClass( \WP_UnitTest_Factory $factory ) {
+		self::$post           = $factory->post->create_and_get();
+		self::$different_post = $factory->post->create_and_get();
+	}
+
+	public function wpTearDownAfterClass() {
```

Comment:
I don't think this teardown is necessary. WordPress should handle cleaning up these fixtures itself.

Reply:
Actually analyzing other `wpTearDownAfterClass` I've to be more radical and delete the post I did create.